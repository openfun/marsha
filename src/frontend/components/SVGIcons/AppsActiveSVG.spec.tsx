import React from 'react';
import renderer from 'react-test-renderer';
import { AppsActiveSVG } from './AppsActiveSVG';

it('renders AppsActiveSVG correctly', () => {
  const AppsActiveSVGSnapshot = renderer
    .create(
      <AppsActiveSVG
        backgroundColor={'#0249a4'}
        baseColor={'white'}
        height={'54'}
        title={'Show/Hide Chat'}
        width={'54'}
      />,
    )
    .toJSON();
  expect(AppsActiveSVGSnapshot).toMatchSnapshot();
});
