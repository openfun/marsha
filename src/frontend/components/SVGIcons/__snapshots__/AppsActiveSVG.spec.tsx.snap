// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AppsActiveSVG correctly 1`] = `
.c0 rect {
  fill: #0249a4;
}

.c0 path {
  fill: #ffffff;
}

<svg
  className="c0"
  height="54"
  viewBox="0 0 54 54"
  width="54"
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    Show/Hide Chat
  </title>
  <g
    fillRule="evenodd"
    stroke="none"
    strokeWidth="1"
  >
    <g
      transform="translate(-1012 -681)"
    >
      <g
        transform="translate(1012 681)"
      >
        <rect
          height="54"
          rx="6"
          width="54"
          x="0"
          y="0"
        />
        <g
          transform="translate(2.5 2.5)"
        >
          <path
            d="M8.333 16.667h8.334V8.333H8.333v8.334zm12.5 25h8.334v-8.334h-8.334v8.334zm-12.5 0h8.334v-8.334H8.333v8.334zm0-12.5h8.334v-8.334H8.333v8.334zm12.5 0h8.334v-8.334h-8.334v8.334zm12.5-20.834v8.334h8.334V8.333h-8.334zm-12.5 8.334h8.334V8.333h-8.334v8.334zm12.5 12.5h8.334v-8.334h-8.334v8.334zm0 12.5h8.334v-8.334h-8.334v8.334z"
            fillRule="nonzero"
          />
        </g>
      </g>
    </g>
  </g>
</svg>
`;
